runtime/onert/backend/cpu/ops/FullyConnectedLayer.h

```diff
@@ -72,7 +72,7 @@ class FullyConnectedLayer : public ::onert::exec::IFunction
 
 #ifdef USE_RUY_GEMV
   uint8_t *_cached_weights = nullptr; // weights to be cached and a key
-  bool _is_weights_freed = false;     // is weights freed?
+// bool _is_weights_freed = false; // TODO Use _is_weights_freed to check weights freed?
```

Comment:
Hmm.. if `EXPERIMENTAL_RUY_FEATURE` is set, it would be build failed.

How about adding?

```c++
#ifdef EXPERIMENTAL_RUY_FEATURE
  bool _is_weights_freed = false; // is_weights_freed
#endif 
```

Reply:
If urgent, I will approve this, and then I'll fix it soon